<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaveApplicationForm</class>
 <widget class="QWidget" name="LeaveApplicationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="formHeadingLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1091</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Leave Form&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeOfLeaveLabel">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Leave Category:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeOfLeaveComboBox">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>180</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Casual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Earned</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Official</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unpaid</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="fromDateLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>252</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;From:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fromDateDateEdit">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="toDateLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>251</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;To:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="toDateDateEdit">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reasonLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Reason:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="leaveReasonTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>1021</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyConfirmButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>640</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
